clang/test/dpct/driver-mem.cu

```diff
@@ -162,6 +162,8 @@ int main(){
 
     // CHECK: dpct::dpct_memcpy(cpy);
     cuMemcpy2D(&cpy);
+    // CHECK: dpct::dpct_memcpy(cpy);
+    cuMemcpy2DUnaligned(&cpy);
```

Comment:
Same as above.

Reply:
To validate the migration rule of cuMemcpy2DUnaligned API [here](https://github.com/oneapi-src/SYCLomatic/blob/9b40b2c9ccf333db559fb7e2448acd12cedc1c44/clang/lib/DPCT/RulesLang/RulesLang.cpp#L5954)

Reply:
Removed it